bot.py

```
@@ -1,36 +1,75 @@
+heartbeat_interval = None
+sequence = None
+
+def send_heartbeat(ws):
+    global sequence
+    while True:
+        if heartbeat_interval:
+            time.sleep(heartbeat_interval / 1000)
+            try:
+                ws.send(json.dumps({"op": 1, "d": sequence}))
+                print(f"💓 Heartbeat (seq: {sequence})")
+            except:
```

Ruff (0.15.2)

[error] Do not use bare `except`

(E722)